gym/lib/gym/options.rb

```diff
@@ -228,6 +228,12 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        UI.user_error!("File not found at path '#{File.expand_path(value)}'") unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :xcode_preference_plist_path,
```

Comment:
Why do we need this option? We usually try to avoid adding options as much as possible, and only add ones that are helpful to a majority of our users 👍